back-end/models/responses-models.js

```diff
@@ -0,0 +1,25 @@
+const db = require("../db/connection");
+
+exports.insertResponse = ({ user_id, fixture_id, response }) => {
+  const insertVals = [user_id, fixture_id, response];
+  return db
+    .query(
+      `
+  DELETE FROM responses WHERE user_id = $1 AND fixture_id = $2;
+  `,
+      [user_id, fixture_id]
+    )
+    .then(() => {
+      return db.query(
+        `
+    INSERT INTO responses (user_id, fixture_id, response)
```

Comment:
Is it necessary to delete and then insert. Can't we achieve the same thing by just using update command?

Reply:
It needs to also work when the response does not already exist